Classes/PKRequestParams.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 
 /// `PKRequestParamsDecorator` used for getting updated request info
 @objc public protocol PKRequestParamsAdapter {
+    var sessionId: String { get set }
```

Comment:
PKRequestParamsAdapter in general doesn't define any relationship to sessionId. I think a more generic approach is needed: add a function paramsUpdated() that takes a Player instance. 
KalturaPlaybackRequestAdapter will use this to get a new sessionId. Other implementations may do other things -- not necessarily related to the sessionId.